packages/dso-toolkit/components/Componenten/dropdown-button/dropdown-button.njk

```diff
@@ -1,6 +1,6 @@
-<div {{ className('dropdown', [open, 'open']) }}>
-  <button type="button" class="btn btn-default dropdown-toggle" id="{{ id }}" aria-haspopup="true" aria-expanded="{% if open %}true{% else %}false{% endif %}">{{ label }}</button>
-  {% if open %}
+<div {{ className('dropdown', [ariaExpanded, 'open']) }}>
```

Comment:
Een dropdown kan een state `open` hebben, en is niet open vanwege `ariaExpanded`: De voorwaarde is andersom: ariaExpanded is true als de dropdown open is.

De correcte naam voor de variable is `open`

Reply:
aangepast